### PR TITLE
Immersive reader inside adaptive card

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by Pawan-MSFT — APPROVED** [review]
Looks correct, approving.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

File: samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs [hunk 1]
```diff
@@ -50,6 +52,8 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
                 await CardActivityAsync(turnContext, true, cancellationToken);
             else if(text.Contains("message"))
                 await MessageAllMembersAsync(turnContext, cancellationToken);
+            else if(text.Contains("immersivereader"))
```
Comment (Wajeed-msft) [inline samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs:55]: We need entry point for to be added from welcome message. 

See this: 
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/642#discussion_r1058286867

**Review by Wajeed-msft — APPROVED** [review]

File: samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs:55]: Yes added it

**Comment by Prithvi-MSFT** [comment]
Build Pipeline Status - All builds passed.
image

**Pull request merged**